src/Compilers/Core/Portable/Emit/EditAndContinue/EmitBaseline.cs

```diff
@@ -70,16 +70,16 @@ internal sealed class MetadataSymbols
             public readonly IReadOnlyDictionary<AnonymousTypeKey, AnonymousTypeValue> AnonymousTypes;
 
             /// <summary>
-            /// In C#, the set of anonymous delegates where the parameter types and return type are
-            /// generic type arguments; in VB, this set is unused and empty.
+            /// In C#, the set of anonymous delegates with their type fully serialized into their name;
+            /// in VB, this set is unused and empty.
             /// </summary>
             public readonly IReadOnlyDictionary<SynthesizedDelegateKey, SynthesizedDelegateValue> AnonymousDelegates;
 
             /// <summary>
-            /// In C#, the set of anonymous delegates where at least one of the parameter types or return type
-            /// is not a valid type argument; in VB, this set is unused and empty.
+            /// In C#, the set of anonymous delegates with unique names generated during compilation;
```

Comment:
>with unique names generated during compilation [](http://example.com/codeflow?start=54&length=46)

This phrase also feels somewhat confusing, because all types have unique names